### ISSUE TITLE
Should core team members be listed as leadership council alumni?

### ISSUE THREAD

**Issue by Manishearth** [rust-lang/team#1626]
I just noticed: old core team members are no longer reachable on the website anywhere.

Since core is the spiritual predecessor of the leadership council, and was a team involving a _significant_ amount of effort, would it make sense to list core team alumni under the council?

**Comment by WaffleLapkin** [comment]
Alternatively we could maybe somehow display now defunct teams.

**Comment by apiraino** [comment]
I agree it's important to show the core team in the acknowledgements of the project. I see under `./teams/archive/` there's a bit of everything: are there other original teams that deserve proper credit? For example what was the [lang reference team](https://github.com/rust-lang/team/blob/master/teams/archive/reference.toml)?

Maybe we could add a static page somewhere in www.r-l.org acknowledging this past work.

Opinions? Any taker for this? 🙂  I might get around it but not soon

**Comment by jieyouxu** [comment]
Just noticed this issue. The alumni core team are now rendered at https://rust-lang.org/governance/archived-teams.html#team-core. (They are not listed as leadership-council alumni specifically though, and are listed as "core team" separately.)

**Comment by Kobzol** [comment]
I think that having them as a separate team is better, as it reflects more accurately the team history (I think that having core members be alumni of leadership council was suggested as a hack, since before we didn't have archived teams on the website).

So I think that this can be closed.

**Comment by Manishearth** [comment]
> I think that having core members be alumni of leadership council was suggested as a hack

I wasn't really suggesting it as a hack. The council does the same type of work; it's more or less the same team. There are a bunch of differences too, but the Rust project has been through many phases of what "core team" means (responsibilities, how the team gets selected, the primary focus, the relative openness of the team, ...) and there are equally-large differences there as well. In the latest iteration of governance reform the team got renamed.

But archived teams is ultimately fine.

https://rust-lang.org/governance/archived-teams.html#team-core

**Comment by Kobzol** [comment]
The council is definitely a spiritual successor of the core team, but on the website we use the actual teams from the team DB, and core and leadership-council are separate entities there, so I think it's fine to separate them.
